determined-ai/determined
# feat: add task tab in workspace

## Description

[WEB-809]

related: #5754, #5792

<img width="1180" alt="image" src="https://user-images.githubusercontent.com/109113616/213813229-01051dfd-ac67-46b8-b4d0-6d03f554f1a5.png">






## Test Plan
- Check if tab (pivot) is visible on both OSS and RBAC enabled
- Check if `userSettings` of task table are independent each other
- Check if API params (request/response) are correctly set (variable name, type etc)
- Check if Task table is correctly working
- Check if code is cuspy





## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket






[WEB-809]: https://determinedai.atlassian.net/browse/WEB-809?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
